hangman.js

```diff
@@ -0,0 +1,95 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Good project overall. I liked the clean design for the page as well as game logic that works well. Good job and keep up the good work!